instr.ml

```diff
@@ -398,12 +404,13 @@ class map = object (m)
       Assert (m#expression e)
     | Stop e ->
       Stop (m#expression e)
-    | Osr {label; cond; target; map} ->
+    | Osr {label; cond; target; var_map; frame_maps} ->
       Osr {
         label;
```

Comment:
ok, sorry I think I escalated this into a bigger argument than necessary. apologies. I'll try to restore the visitor as it was.